serde/msgpack.py

```diff
@@ -26,9 +32,17 @@ def deserialize(cls, s, raw=False, use_list=False, **opts):
 
 
 def to_msgpack(obj: Any, se=MsgPackSerializer, **opts) -> bytes:
+    "If `ext_dict` option is specified, `obj` is encoded as a `msgpack.ExtType`"
```

Comment:
Let's make it a docstring!
```suggestion
    """
    If `ext_dict` option is specified, `obj` is encoded as a `msgpack.ExtType`
    """
```